Weather/Weather/Controllers/WeatherController.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  ViewController.swift
+//  Weather
+//
+//  Created by Sergii Lisnyi on 5/15/18.
+//  Copyright © 2018 Sergii Lisnyi. All rights reserved.
+//
+
+import UIKit
+
+//import CoreLocation
+//import MapKit
+
+class WeatherController: UIViewController {
+
+    var model = WeatherService()
```

Comment:
Consider to refactor `WeatherService`
Store it in PageViewController, use for loading data only.
Store forecasts together with cities as optionals.
When getting onto city screen, ask delegate for forecast.
Delegate either returns already loaded data, or loads and returns - both in closures